Dexterous-Rambutan/battle-code
# Hooked up to evaluated redisQueue, and now emits a socket event to the appropriate socket client

#### What's this PR do?

Hooked up socket emitter to blockPopping of evaluated redisQueue
#### What are the important parts of the code?

./responseRedis
#### How should this be tested by the reviewer?
1. Run npm install
2. Run redis-server -&
3. Run redis-cli
4. Run node server/server.js
5. Inside redis-cli enter: LPUSH rQueue 'helloWorld'
6. Expect server console log to say "Successfully popped from redisQueue ['rQueue', 'helloWorld']"
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

References #30
#### Screenshots (if appropriate)
